tests/post_training/conftest.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import numpy as np
+from pathlib import Path
+
+def pytest_addoption(parser):
+    parser.addoption("--data", action="store", default="/mnt/datasets/imagenet/val")
+    parser.addoption("--output", action="store", default="./tmp/")
+
+def pytest_configure(config):
+    config.test_results = []
+
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
+def pytest_runtest_makereport(item, call):
+    outcome = yield
+    result = outcome.get_result()
+
+    if result.when == 'call':
+        table = item.config.test_results
+        header = ["Model name", "FP32 top-1", "Torch INT8 top-1", "ONNX INT8 top-1", "OV INT8 top-1",
+            "FP32 FPS", "Torch INT8 FPS", "ONNX INT8 FPS", "OV INT8 FPS"]
```

Comment:
Looks fragile. I think you should rather set up a `BenchmarkResult` dataclass with fields such as `model`, `framework`, `result_type` (accuracy or performance) and `value`; you can also add the `succeeded` and `error_msg` fields for error handling if you want to see it in the report. You can then group the results and make a table in this function. This will remove the need for awkward `result.append([model_name, error, "-", "-", "-", "-", "-", "-", "-"])` stuff where one needs to put the exactly correct number of `"-"` elements in the other file to always correspond to the header length in this file.

Reply:
Won't fix as minor.